include/dnnl_types.h

```diff
@@ -2447,6 +2447,22 @@ typedef enum {
     /// Intel AMX with 8-bit integer and bfloat16 support
     /// (initial support)
     dnnl_cpu_isa_avx512_core_amx = 0x3e7,
+
+    /// AARCH64 Advanced SIMD & floating-point
+    dnnl_cpu_isa_asimd = 0x1000000,
+
+    /// AARCH64 SVE 128 bits
+    dnnl_cpu_isa_sve_128 = 0x8000000,
+
+    /// AARCH64 SVE 256 bits
+    dnnl_cpu_isa_sve_256 = 0x18000000,
+
+    /// AARCH64 SVE 384 bits
+    dnnl_cpu_isa_sve_384 = 0x28000000,
+
+    /// AARCH64 SVE 512 bits
+    dnnl_cpu_isa_sve_512 = 0x38000000,
+
```

Comment:
There is no problem to define cpu_isa_type for AArch64 after an RFC is accepted.
Instead of defining dnnl_cpu_isa_t for AArch64 in `dnnl_types.h` and `dnnl.hpp`, 
tmp_dnnl_cpu_isa_t is defined in `cpu/aarch64/cpu_isa_traits.hpp`. Is this OK?
Here is sample implementation https://github.com/kawakami-k/oneDNN/commit/0f76a9873a35b1938e09f4e84d95f1abc449038e

Reply:
Yes, that is fine with me. Thank you, and sorry for the disturbance.

A minor suggestion for kawakami-k@0f76a98:
1. No need to have `tmp_dnnl_cpu_isa_t`, let it be just unnamed enum
2. No need to use x64 named values, it's better to just use the values you prefer.

In other words (I put some values for enums, feel free to change to the appropriate ones):

``` cpp
// TODO: The following values should be part of dnnl_cpu_isa_t enum
//       for Aarch64 configuration. Temporary keep them here, until
//       architecture macro (e.g. DNNL_AARCH64) is put in dnnl_config.h
enum {
    /// AARCH64 Advanced SIMD & floating-point
    dnnl_cpu_isa_asimd = (1 << 0),

    /// AARCH64 SVE 128 bits
    dnnl_cpu_isa_sve_128 = (1 << 1),

    /// AARCH64 SVE 256 bits
    dnnl_cpu_isa_sve_256 = (1 << 2),

    /// AARCH64 SVE 384 bits
    dnnl_cpu_isa_sve_384 = (1 << 3),

    /// AARCH64 SVE 512 bits
    dnnl_cpu_isa_sve_512 = (1 << 4),
};
```

Also, `src/common/dnnl_debug_autogenerated.cpp` is auto-generated (using `scripts/generate_dnnl_debug.py` python script), so the change there could be easily lost. The good news is that this function is not used anywhere so far. So, I suggest we do not change this file keeping a technical debt for now, and fix it when we put Aarch64 ISAs into the public header.

Reply:
I removed dnnl_cpu_isa_* for AArch64 from public headers and dnnl_debug_autogenerated.cpp
https://github.com/oneapi-src/oneDNN/pull/850/commits/c63f0995942c801a6d91cecc84f6d71d7047e2aa

Reply:
LGTM, thank you!